utils/rpms/daos.spec

```diff
@@ -234,8 +234,13 @@ PREFIX="%{?_prefix}"
 mkdir -p %{?buildroot}/%{_sysconfdir}/ld.so.conf.d/
 echo "%{_libdir}/daos_srv" > %{?buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
 mkdir -p %{?buildroot}/%{_unitdir}
+%if (0%{?rhel} == 7)
```

Comment:
why not < 8? we don't necessarily support earlier versions but it seems more intuitive

Reply:
Because systemd didn't exist on < 7.

Reply:
good point, thanks